src/optionals/argparse/argparse-source.h

```diff
@@ -0,0 +1,133 @@
+#define DICTU_ENV_SOURCE "import Argparse; \
```

Comment:
We will want `DICTU_ARGPARSE_SOURCE` here but using the generate script (once we add the argparse path should resolve this)